cookbook-react/recipes/inputs/input-3/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import cn from 'classnames';
+
+import './styles.scss';
+
+interface Props {
+  label?: string;
+  disabled?: boolean;
+  error?: string;
+}
+
+function Input({ label, disabled, error }: Props) {
+  return (
+    <div id="input-3" className="input-container">
+      <div className="input-wrapper">
+        <input
+          className={cn("input-base", { "input-error": !!error, "input-disabled": disabled })}
+          placeholder=" "
+          disabled={disabled}
+        />
+        <label className={cn("input-label", { "label-disabled": disabled })}>{label}</label>
```

Comment:
Check what @BJordi said about the order of this tags and the `id` and `for` attributes 😄

Reply:
The problem with changing the order of the tags is that i didnt have a way to change the label after input had trigger with `active` css selector if i had it before the input tag.